pkg/registry/apis/dashboard/search.go

```diff
@@ -303,6 +303,20 @@ func (s *SearchHandler) DoSearch(w http.ResponseWriter, r *http.Request) {
 		}}
 	}
 
+	// The ids filter
+	ids, ok := queryParams["uid"]
```

Comment:
should we use "name" rather than "uid"?  or even "id" would be less confusing.

Reply:
technically we are filtering by `name`.  it was a bit confusing though to me ( and probably others ) that name here gets converted to id.  so `id` makes more sense to me